pages/en/network/benefits.mdx

```diff
@@ -86,7 +76,7 @@ No contracts. No monthly fees. Only pay for the queries you use—with an averag
 
 <sup>‡</sup>using the max query budget function in the budget billing tab, while maintaining high quality of service
 
-Estimated costs only for Ethereum Mainnet subgraphs— costs are even higher when self hosting a graph node on other networks.
+Estimated costs are only for Ethereum Mainnet subgraphs — costs are even higher when self hosting a graph node on other networks.
```

Comment:
Same comment as above about "a graph node". Not sure it should be lowercase.

Reply:
Got them all fixed!